src/import/animatron-importer.js

```diff
@@ -744,8 +748,16 @@ Import.fill = function(src) {
  */
 Import.stroke = function(src) {
     if (!src) return null;
-    return Brush.stroke(is.arr(src[1]) ? Import.grad(src[1])
-                                       : src[1], // paint
+    var fill;
+    if (is.str(src[0])) {
+        fill = src[0];
+    } else if (is.arr(src[0])) {
+        if (is.arr(src[0][0])) {
+            fill = Import.grad(src[0]);
+        }
+        fill = Import.pattern(src[0]);
+    }
+    return Brush.stroke(fill, // paint
```

Comment:
In removed lines above, first argument for `Brush.stroke` was `src[1]` for both array-case and non-array-case. Why was it changed to `src[0]` here, while `src[0]` is a width, as the line below states?